Xamarin.PropertyEditing.Mac/Controls/PropertyEditorControl.cs

```diff
@@ -146,6 +112,66 @@ public sealed override void ViewDidChangeEffectiveAppearance ()
 
 			AppearanceChanged ();
 		}
+
+		public void OnNextResponderRequested ()
+		{
+			if (TableView != null) {
+				nint row = TableView.RowForView (this) + 1;
+
+				NSView view;
+				PropertyEditorControl ctrl = null;
+
+				for (; row < TableView.RowCount; row++) {
```

Comment:
This will query `TableView.RowCount` on every loop iteration, which is an objc pinvoke. I think you want to put it above the loop:
```suggestion
				var rowCount = TableView.RowCount;
				for (; row < rowCount; row++) {
```

Reply:
fixed